app/views/applications/show.html.erb

```diff
@@ -0,0 +1,62 @@
+<br>
+<h1>Name of Applicant: <%= @application.name %></h1>
+
+<section id="applicant-info">
+<p>
+  Applicant Address:
+  <%= @application.street_address %>,
+  <%= @application.city %>, <%= @application.state %>, <%= @application.zip_code %>
+</p>
+<p>Description of why the applicant says they'd be a good home for this pet(s): <%= @application.description %></p>
+</section>
+
+<br>
+
+<section id="pets-applied-for">
+<h2>Pets Applied for:</h2>
+<ul>
+  <% @application.pets.each do |pet| %>
+    <li><%= link_to pet.name, "/pets/#{pet.id}" %></li> 
+  <% end %>
+</ul>
+</section>
+
+<section id="app-status">
+<p> Application's status: <%= @application.status %></p>
+</section>
+<br>
+
+<section id="application-submit">
+  <% if @application.pets.count != 0 %>
+    <%= form_with url: "/applications/#{@application.id}", method: :patch, data: { turbo: false } do |form| %>
+      <br><%= form.label :adopt_description, "Why would you make a good owner for these pet(s)?" %>
+      <%= form.text_field :adopt_description %>
+      <%= form.submit "Submit Application" %>
+    <% end %>
+  <% end %>
+</section>
+<br>
+
+<section id="pet-search">
+<% if @application.status == "In Progress" %>
+  <h3>Add a Pet to this Application</h3>
+  <%= form_with url: "/applications/#{@application.id}", method: :get, data: { turbo: false } do |form| %>
+    <%= form.hidden_field :search, value: "pet" %>
+    <%= form.label :pet_name, "Search for Pets" %>
+    <%= form.text_field :pet_name %>
+    <%= form.submit "Submit" %>
+  <% end %>
+<% end %>
+</section>
+
+<br>
+
+<section id="adopt-a-pet">
+<% if !@search_pets.nil? && @search_pets.any? %>
+  <% @search_pets.each do |pet| %>
+    <%= form_with url: "/application_pets?pet_id=#{pet.id}&application_id=#{@application.id}", method: :post, data: { turbo: false } do |form| %>
```

Comment:
Similar idea here -- this is a RESTful route, but if you wanted to, you could also put the IDs in the path and it would still be considered RESTful as long as you're mentioning both applications and pets in the path to indicate the association between these resources. 
example: "/applications/:application_id/pets/:pet_id"